### ISSUE TITLE
Is the distribution directories structure correct?

### ISSUE THREAD

**Issue by dfumagalli** [vqmod/vqmod#1]
Hello,

I think there might be something unusual going on.

It all started with a number of vQModerator users reporting total destruction of their setups when upgrading to vQMod 2.5. I though it was just the usual stuff: bad hosting with wrong permissions, inexperienced users and so on.

But now I want to upgrade to 2.5 myself and I have noticed "peculiarities" I don't understand.

Downloading the zip and expanding it, it reveals an unusual "platform" directory. Under that directory is the install.php file. At the same time it's missing from the location I expect it to be: vqmod\install.

I have read the source code: the /platforms/opencart/install/index.php file requires /vqmod/install/ugrsr.class.php (as it has done in the past)... too bad index.php is under a totally different branch so it won't find it.

Basically, it's like the vqmod directory structure is split in two branches and I don't understand how the install can work with half installer under platform and the required files under vqmod and platform branch requires want to include vqmod/install stuff from their platform directory.

Could you please explain me how to make the whole thing work? I have a strong suspicion that if I merge everything under a vqmod folder and move the install.php in the vqmod/install directory things could work. But then, the vqmod xml is under platform as well whereas on my installation is under vqmod.
